app/models/pet_application.rb

```diff
@@ -1,4 +1,5 @@
 class PetApplication < ApplicationRecord
+  validates :status, presence: true
```

Comment:
It will be good to have these validations added for the columns we manually to the tables